src/test/java/racingcar/model/CarTest.java

```diff
@@ -43,4 +53,14 @@ void check_move_when_value_under_the_condition_four() {
         car.move(3);
         assertThat(car.getPosition()).isEqualTo(0);
     }
+
+
+    @Test
+    @DisplayName("Car 이름이 5자 초과시 exception 발생")
+    void expected_exception_when_name_length_over_five(){
+        assertThatThrownBy(()->{
+            Car car = new Car("test11");
+        }).isInstanceOf(RuntimeException.class)
+                .hasMessage("자동차 이름은 5자를 초과할 수 없습니다.");
```

Comment:
Car 내부에서 해당 오류 메세지를 static value 로 만들어주면 아래 처럼 쓸 수 있을 것 같아요 :) 
```suggestion
                .hasMessage(Car.NAME_LENGTH_EXCEPTION_MESSAGE);
```